data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -302,26 +314,42 @@ def _create_downloadable_data(
                 ).astype("Int64")
                 df[column] = df_int
 
-            elif column in constants.FEMA_ROUND_NUM_COLUMNS:
+            elif column_type_dict[column] == "loss_rate_percentage":
                 # Convert loss rates by multiplying by 100 (they are percents)
                 # and then rounding appropriately.
                 df_100 = df[column] * 100
                 df[column] = floor_series(
                     series=df_100.astype(float64),
-                    number_of_decimals=constants.TILES_FEMA_ROUND_NUM_DECIMALS,
+                    number_of_decimals=config_object["rounding_num"][
+                        "loss_rate_percentage"
+                    ],
                 )
 
-            else:
-                # Round all other floats.
+            elif column_type_dict[column] == "float":
+                # Round the floats.
                 df[column] = floor_series(
                     series=df[column].astype(float64),
-                    number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
+                    number_of_decimals=config_object["rounding_num"]["float"],
                 )
```

Comment:
can we add something like: 

```
else: 
    raise ValueError(f"Unrecognized type: `{column_type_dict[column]}`")

Reply:
bump

Reply:
still bumping this @esfoobar-usds -- otherwise we'll swallow the error and not realize something is amiss

Reply:
Crap.. I swore I made this change! Done now for realsies!